cypress/integration/4_findings.spec.js

```diff
@@ -0,0 +1,170 @@
+    cy.get(`[data-test-subj="close-finding-details-flyout"]`).then(($el) => {
+      cy.get($el).click({ force: true });
+    });
+
+    // wait for toasts to clear - in this case, timeout will not work.  Icon cannot be found behind error toasts.
+    cy.wait(10000);
+
+    // Click View details icon
+    cy.get(`[data-test-subj="view-details-icon"]`).click({ force: true });
+
+    // Confirm flyout contents
+    cy.contains('Finding details');
+    cy.contains('Rule details');
+    cy.contains('Search Findings').should('not.exist');
+
+    // Close Flyout
+    cy.get(`[data-test-subj="close-finding-details-flyout"]`).then(($el) => {
+      cy.get($el).click({ force: true });
+    });
+  });
+
+  it('allows user to view details about rules that were triggered', () => {
+    // open Finding details flyout via finding id link
+    cy.wait(5000);
+    cy.get(`[data-test-subj="view-details-icon"]`).click({ force: true });
+
+    // Second rule details - open
+    cy.get('.euiAccordion__button')
+      .contains('Setting Change in Windows Firewall with Advanced Security')
+      .click({ force: true });
+
+    // Confirm content
+    cy.contains('Setting have been change in Windows Firewall');
+    cy.contains('Low');
+    cy.contains('Windows');
+    cy.contains('cypress-test-windows');
+
+    ruleTags.forEach((tag) => {
+      cy.contains(tag);
+    });
+
+    // Close Flyout
+    cy.get(`[data-test-subj="close-finding-details-flyout"]`).then(($el) => {
+      cy.get($el).click({ force: true });
+    });
+
+    // need to wait for error toasts to dissipate, neither icon nor finding id are clickable without this
+    cy.wait(5000);
+
+    // open Finding details flyout via icon button
+    cy.get(`[data-test-subj="view-details-icon"]`).click({ force: true });
+
+    cy.get('button', { timeout: 1000 });
+    cy.get('.euiAccordion__button').contains('USB Device Plugged').click({ force: true });
+
+    // Confirm content
+    cy.contains('Documents');
+    cy.contains('Detects plugged USB devices');
+    cy.contains('Low');
+    cy.contains('Windows');
+    cy.contains('cypress-test-windows');
+
+    ruleTags.forEach((tag) => {
+      cy.contains(tag);
+    });
+  });
+
+  // TODO - upon reaching rules page, trigger appropriate rules detail flyout
+  it('takes user to rules page when rule name inside accordion drop down is clicked', () => {
+    // Click rule link
+    cy.get('a')
+      .contains('USB Device Plugged')
+      .invoke('removeAttr', 'target')
+      .click({ force: true });
+
+    // Confirm destination reached
+    cy.url().should(
+      'eq',
+      'http://localhost:5601/app/opensearch_security_analytics_dashboards#/rules'
+    );
+  });
+
+  after(() => {
+    // Visit Detectors page
+    cy.visit(`${Cypress.env('opensearch_dashboards')}/app/${PLUGIN_NAME}#/detectors`, {
+      timeout: 5000,
+    });
+
+    cy.url().should(
+      'include',
+      'http://localhost:5601/app/opensearch_security_analytics_dashboards#/detectors'
```

Comment:
Let's just check for `opensearch_security_analytics_dashboards#/detectors`?

Reply:
Sure.